src/main/java/io/jenkins/plugins/appcenter/AppCenterRecorder.java

```diff
@@ -115,7 +116,7 @@ private boolean uploadToAppCenter(Run<?, ?> run, FilePath filePath, TaskListener
         final PrintStream logger = taskListener.getLogger();
 
         try {
-            final AppCenterServiceFactory appCenterServiceFactory = new AppCenterServiceFactory(getApiToken(), getBaseUrl());
+            final AppCenterServiceFactory appCenterServiceFactory = new AppCenterServiceFactory(getApiToken(), getBaseUrl(), Jenkins.getInstanceOrNull());
```

Comment:
`Jenkins::get` is preferred instead of `Jenkins::getInstanceOrNull`.

Reply:
I was somehow confused as I first used `Jenkins::getInstance` which was marked as Deprecated. ``::get` actually seems to be the right choice. Changed.